go.mod

```diff
@@ -32,8 +32,6 @@ require (
 	k8s.io/apimachinery v0.26.1
 )
 
-replace gopkg.in/yaml.v3 => github.com/colega/go-yaml-yaml v0.0.0-20220720070545-aaba007ebc22
```

Comment:
Are the changes made in this fork already upstreamed?

Reply:
I'm now wondering how this affects helmfile's overall functionality... which of our dependencies are still using yaml.v3? 🤔

Reply:
vals @mumoshu

Reply:
@yxxhero Thanks! Once this is merged, shall we make vals use the combination of yaml.v2 and goccy-goyaml too so that the syntaxes like `<<: [*foo, *bar]` works uniformly between helmfile.yaml and yaml embedded in a secret retrieved via vals? (Note that vals uses yaml.v3 to "extract a value inside the secret yaml obj" using the `#/yaml/path/to/key` syntax.